AUTHORS

```diff
@@ -0,0 +1 @@
+Open Robotics
```

Comment:
Add youself and remember newline at end of file.

Reply:
Ok